module/importer-ergonode-1/src/Infrastructure/Reader/ErgonodeAttributeReader.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\ImporterErgonode\Infrastructure\Reader;
+
+use Ergonode\ImporterErgonode\Infrastructure\Model\AttributeModel;
+
+class ErgonodeAttributeReader extends AbstractErgonodeReader
+{
+    /**
+     * @throws \JsonException
+     */
+    public function read(): ?AttributeModel
+    {
+        $item = null;
+
+        while ($this->records->valid()) {
+            $record = $this->records->current();
+
+            if (null === $item) {
+                $item = new AttributeModel(
+                    $record['_id'],
```

Comment:
Wondering wouldn't it be cleaner to use models with serializer configuration? So it'd be shared and less such mapping should be performed manually?

Reply:
Yes i think that will be great idea, but in first iteration we should check compatibility and wait for edge cases. This solution is easier to change and debug.